src/lib.rs

```diff
@@ -0,0 +1,28 @@
+#[macro_use]
+extern crate lazy_static;
+
+mod compiler;
+mod execute_command_line;
+mod rust_helpers;
+
+pub use compiler::command_line_parser::parse_command_line;
```

Comment:
In the Typescript code, all of the helpers etc from different files live under a "flat" `namespace ts` so having them be exposed as top-level names under the crate seems like the equivalent?

Reply:
Yep, and I think that's a pretty typical pattern in rust either way.